definitions/checks/unread_mail.rb

```diff
@@ -0,0 +1,46 @@
+class Checks::UnreadMail < ForemanMaintain::Check
+  metadata do
+    label :unread_mail
+    description "Check for mails in root's mailbox."
+    tags :unread_mail
```

Comment:
@kgaikwad We are tagging this check with unread_mail...Currently this  check won't run before upgrade unless user run it explicitly ( foreman-maintain health check --label unread-mail ). So I think to this need to add in scenario definitions https://github.com/theforeman/foreman_maintain/blob/master/definitions/scenarios/upgrade_to_satellite_6_3.rb#L23 

add_step(Checks:::UnreadMail.new)

Reply:
@ntkathole ,
It's not intended to be a pre/post upgrade checks. That's why tagged as `unread_mail` so that any user can run it explicitly whenever required.